### ISSUE TITLE
New algorithms

### ISSUE THREAD

**Issue by jtesta** [arthepsy/ssh-audit#38]
I found some non-standard SSH algorithms in use out in the wild that are not currently supported:
```
des-cbc-ssh1
blowfish-ctr
hmac-sha256
hmac-sha256@ssh.com
hmac-sha2-384
```

**Comment by arthepsy** [comment]
Do You have more information, e.g., where they originate from?

**Comment by jtesta** [comment]
Nope.  This is just from observing the debug log of what people are scanning with my online front-end (https://www.sshaudit.com/).

**Comment by egberts** [comment]
FYI https://soatok.blog/2022/05/19/guidance-for-choosing-an-elliptic-curve-signature-algorithm-in-2022/
